qt/python/mantidqt/mantidqt/widgets/codeeditor/execution.py

```diff
@@ -34,39 +32,15 @@
 FILE_ATTR = "__file__"
 
 
-def _get_imported_from_future(code_str):
-    """
-    Parse the given code and return a list of names that are imported
-    from __future__.
-    :param code_str: The code to parse
-    :return list: List of names that are imported from __future__
-    """
-    future_imports = []
-    code_str = code_str.encode("utf-8")
-    for node in ast.walk(ast.parse(code_str)):
-        if isinstance(node, ast.ImportFrom):
-            if node.module == "__future__":
-                future_imports.extend([import_alias.name for import_alias in node.names])
-                break
-    return future_imports
-
-
-def get_future_import_compiler_flags(code_str):
+def get_future_import_compiler_flags(code_str) -> int:
```

Comment:
are you planning getting rid of this function? It does serve any purpose now, right?

Reply:
It is mostly documentation. I guess I can drop it too